x-pack/plugins/security_solution/public/common/components/and_or_badge/__examples__/index.stories.tsx

```diff
@@ -14,26 +14,21 @@ import { AndOrBadge } from '..';
 const sampleText =
   'Doggo ipsum i am bekom fat snoot wow such tempt waggy wags floofs, ruff heckin good boys and girls mlem.  Ruff heckin good boys and girls mlem stop it fren borkf borking doggo very hand that feed shibe, you are doing me the shock big ol heck smol borking doggo with a long snoot for pats heckin good boys. You are doing me the shock smol borking doggo with a long snoot for pats wow very biscit, length boy. Doggo ipsum i am bekom fat snoot wow such tempt waggy wags floofs, ruff heckin good boys and girls mlem.  Ruff heckin good boys and girls mlem stop it fren borkf borking doggo very hand that feed shibe, you are doing me the shock big ol heck smol borking doggo with a long snoot for pats heckin good boys.';
 
+const withTheme = (storyFn: () => ReactNode) => (
+  <ThemeProvider theme={() => ({ eui: euiLightVars, darkMode: true })}>{storyFn()}</ThemeProvider>
+);
+
 storiesOf('components/AndOrBadge', module)
-  .add('and', () => (
-    <ThemeProvider theme={() => ({ eui: euiLightVars, darkMode: true })}>
-      <AndOrBadge type="and" />
-    </ThemeProvider>
-  ))
-  .add('or', () => (
-    <ThemeProvider theme={() => ({ eui: euiLightVars, darkMode: true })}>
-      <AndOrBadge type="or" />
-    </ThemeProvider>
-  ))
+  .addDecorator(withTheme)
+  .add('and', () => <AndOrBadge type="and" />)
```

Comment:
We don't need to do this now, but FYI: there's a storybook addon (`storybook/addon-knobs`) that allows you to define stories with interactive props. This helps to keep the navigation simpler and align closer to one story per component (instead of one story per props permutation).

Reply:
Have you worked with mocking hooks in storybook. There's a bunch of different things I'd love to play around with on it. I'll take a look at knobs. Thanks!